package.json

```
@@ -13,6 +13,7 @@
     "serve": "rollup -c rollup.config.js -w",
     "build": "rollup -c",
     "test": "jest",
+    "test-codecov": "jest --ci --coverage && codecov",
     "update-snapshot": "jest --updateSnapshot"
```

사실 codecov은 local에선 실행되지 않을 명령어라.. 아래 같이 바꿔봐도 좋을 것 같습니다~! (update-snapshot은 덩달아 변경)

```suggestion
    "test:coverage": "jest --coverage",
    "test:snapshot": "jest --updateSnapshot",
    "codecov": "codecov"
    또는
    "codecov": "npm run test:coverage && codecov"
```

---

생각해보니 그렇네요!! 저는 local에서도 보려는 건가?라는 생각은 했었는데 실직적으로 console로 많이 보지 그래프나 그런걸로 보지는 않겠네요!

---

local에서 그래프로 보고 싶으면 `npm run test:coverage`를 실행하면 됩니다~! ㅎㅎ